pelican/plugins/pandoc_reader/test_pandoc_reader.py

```diff
@@ -0,0 +1,1157 @@
+"""Tests for pandoc-reader plugin."""
+# pylint: disable=too-many-lines
```

Comment:
If `pylint` warns about too many lines, and GitHub won't even render the diff without users explicitly tapping a button to load the diff… then maybe this file does indeed have too many lines? Do you think it make make sense to split this into separate files, perhaps one file per test class?